build.sbt

```diff
@@ -8,6 +8,8 @@ lazy val commonSettings = Seq(
   libraryDependencies ++= dependencies.Testing ++ dependencies.CompilerPlugins,
   test in assembly := {},
   assemblyMergeStrategy in assembly := {
+    case PathList("META-INF", "maven", "org.webjars", "swagger-ui", "pom.properties") =>
+      MergeStrategy.singleOrError
```

Comment:
How swagger ui got into jars at all? We don't use it, do we?

Reply:
fix